exporter/collector/config.go

```diff
@@ -113,6 +114,12 @@ type MetricConfig struct {
 	EnableSumOfSquaredDeviation bool `mapstructure:"sum_of_squared_deviation"`
 }
 
+type ImpersonateConfig struct {
```

Comment:
Can you add a comment for this struct with basically the same content as in your PR title?